react/training-project/src/app/components/Game/styles.scss

```diff
@@ -0,0 +1,9 @@
+.game {
+  display: flex;
+  flex-direction: row;
```

Comment:
Are you overriding this prop? If not, then, remember row is the default value for flex-direction.